apps/sms/js/thread_list_ui.js

```diff
@@ -498,7 +505,9 @@ var ThreadListUI = {
     var removeWhenEmpty = empty ? 'remove' : 'add';
 
     ThreadListUI.noMessages.classList[removeWhenEmpty]('hide');
+    ThreadListUI.settingsButton.classList[removeWhenEmpty]('hide');
     ThreadListUI.container.classList[addWhenEmpty]('hide');
+    ThreadListUI.optionsButton.classList[addWhenEmpty]('hide');
```

Comment:
I like "threadlist-is-empty" because it represents a state with the "is" word. It's a convention. But I don't have a strong feeling.